flowrs-build-designer/repository/modules/packages.ts

```diff
@@ -83,9 +84,26 @@ class PackagesModule extends FetchFactory {
             }
         }
 
-        return packageMap
+        return this.filterInActive(packageMap,currentActive)
     }
 
+    private filterInActive(map: Map<string, TypeDefinition>,currentActive: string[]) {
+        const keysToRemove: string[] = [];
```

Comment:
sollte eher filterInactive heißen stat filter in active

Reply:
Korrekt erledigt.